src/endpoint/endpoint.js

```diff
@@ -112,7 +112,8 @@ async function main(options = {}) {
         // the primary just forks and returns, workers will continue to serve
         fork_count = options.forks ?? config.ENDPOINT_FORKS;
         const metrics_port = options.metrics_port || config.EP_METRICS_SERVER_PORT;
-        if (fork_utils.start_workers(metrics_port, fork_count)) return;
+        const workers_were_started_from_primary = await fork_utils.start_workers(metrics_port, fork_count);
```

Comment:
can we have something like `is_workers_started_from_primary`?

Reply:
Yes, I will change it, thanks.